prime-router/src/main/kotlin/Report.kt

```diff
@@ -162,11 +159,7 @@ data class ReportStreamFilterResult(
  * unique id and name as well as list of sources for the creation of the report.
  */
 class Report : Logging {
-    enum class Format(
-        val ext: String,
-        val mimeType: String,
-        val isSingleItemFormat: Boolean = false,
-    ) {
+    enum class Format(val ext: String, val mimeType: String, val isSingleItemFormat: Boolean = false) {
```

Comment:
I know this is nitpicky and I wasn't going to say anything about it, but it looks like you just changed this parameter list from being vertical to being horizontal and in another spot I'll call out below it looks like the only change is from horizontal to vertical.

Reply:
ah yeah that's my auto formatter at work. I'll got back and try to fix it

Reply:
Noooo not the auto formatter changing some to horizontal and some to vertical 😭